pylzy/lzy/api/v1/remote/workflow_service_client.py

```diff
@@ -79,7 +80,12 @@ class StderrMessage:
     data: str
 
 
-Message = Union[StderrMessage, StdoutMessage]
+@dataclass
+class Message:
+    msg: Union[StderrMessage, StdoutMessage]
```

Comment:
I really do not like `Union[StderrMessage, StdoutMessage]`

It have to be 
```
@dataclass
class BaseMessage(ABC):
   data: str
   offset: int

class StdoutMessage(BaseMessage):
    pass
```

or somethink like this.
We can ans we shoild use OOP here because we can.

Reply:
And class Message which includes another Message - is just terrible semantics